### PR TITLE
feedback_system created in which action cabel used

### PR DESCRIPTION
**Pull request by Ridham151101**

### CODE REVIEW

File: feedback_system/app/controllers/admin/feedbacks_controller.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
+class Admin::FeedbacksController < ApplicationController
+  def index
+    @feedbacks = Feedback.where("rating < 3").order(created_at: :desc)
+  end
+
+  def show
+    @feedback = Feedback.find(params[:id])
```
Comment (bacancy-urmit) [inline feedback_system/app/controllers/admin/feedbacks_controller.rb:7]: create a common method to find feedback and call it in before_filter

File: feedback_system/app/controllers/admin/feedbacks_controller.rb [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  end
+
+  def update
+    @feedback = Feedback.find(params[:id])
```
Comment (bacancy-urmit) [inline feedback_system/app/controllers/admin/feedbacks_controller.rb:11]: create a common method to find feedback and call it in before_filter

File: feedback_system/app/controllers/feedbacks_controller.rb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+class FeedbacksController < ApplicationController
+  def new
+    @feedback = Feedback.new
+  end
+
+  def create
+    @feedback = current_user.feedbacks.build(feedback_params)
+    @feedback.user_id = current_user.id
```
Comment (bacancy-urmit) [inline feedback_system/app/controllers/feedbacks_controller.rb:8]: if you have written current_user.feedbacks.build(feedback_params)
then why are you assigning user_id again?

File: feedback_system/app/controllers/feedbacks_controller.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+    if @feedback.save
+      if @feedback.rating < 3
+        ActionCable.server.broadcast("feedbacks_channel", @feedback.as_json)
```
Comment (bacancy-urmit) [inline feedback_system/app/controllers/feedbacks_controller.rb:11]: write it like below
ActionCable.server.broadcast("feedbacks_channel", @feedback.as_json) if  @feedback.rating < 3

**Review by bacancy-urmit — COMMENTED** [review]
Please address all the comments

File: feedback_system/app/controllers/admin/feedbacks_controller.rb (hunk 1 above)
Comment (Ridham151101) [reply feedback_system/app/controllers/admin/feedbacks_controller.rb]: created common method set_feedback and call it befor_action in admin/feedbacks_controller.rb

File: feedback_system/app/controllers/admin/feedbacks_controller.rb (hunk 2 above)
Comment (Ridham151101) [reply feedback_system/app/controllers/admin/feedbacks_controller.rb]: created common method set_feedback and call it befor_action in admin/feedbacks_controller.rb

File: feedback_system/app/controllers/feedbacks_controller.rb (hunk 3 above)
Comment (Ridham151101) [reply feedback_system/app/controllers/feedbacks_controller.rb]: removed current_user.id line and it works.

File: feedback_system/app/controllers/feedbacks_controller.rb (hunk 4 above)
Comment (Ridham151101) [reply feedback_system/app/controllers/feedbacks_controller.rb]: remove if condition and write in one line with if condition

**Pull request closed without merging**